modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/view.jsp

```diff
@@ -31,7 +31,26 @@ ContentDashboardAdminManagementToolbarDisplayContext contentDashboardAdminManage
 				size="<%= StringPool.BLANK %>"
 			>
 				<h2 class="sheet-title">
-					<%= contentDashboardAdminDisplayContext.getAuditGraphTitle() %>
+					<clay:content-row>
+						<clay:content-col
+							expand="<%= true %>"
+						>
+							<%= contentDashboardAdminDisplayContext.getAuditGraphTitle() %>
+						</clay:content-col>
+
+						<clay:content-col>
+							<span class="lfr-portal-tooltip" title="<%= LanguageUtil.get(request, "configure-chart") %>">
+								<clay:button
+									borderless="<%= true %>"
+									cssClass="component-action"
+									displayType="secondary"
+									icon="cog"
+									onClick="<%= portletDisplay.getURLConfigurationJS() %>"
```

Comment:
I always forget about this, which is terrifying 😂 

https://github.com/liferay/liferay-portal/blob/master/portal-web/docroot/html/portal/render_portlet.jsp#L455-L505

It is what it is 😂

Reply:
OMG 😂